docs/_static/css/versionwarning.css

```diff
@@ -0,0 +1,37 @@
+#version-warning-banner {
+  margin: 10px 0;
+  padding: 5px 10px;
+  border-radius: 4px;
+
+  letter-spacing: 1px;
+  color: #fff;
+  text-shadow: 0 0 2px #000;
+  text-align: center;
+  font-size: 0.9em;
+
+  background: #004b6b repeating-linear-gradient(
+    135deg,
+    transparent,
+    transparent 56px,
+    rgba(255, 255, 255, 0.2) 56px,
+    rgba(255, 255, 255, 0.2) 112px
```

Comment:
I think the stripes and the padding could be tweaked a little to improve the readability. (Maybe its just me / my display)

<img width="962" alt="screen shot 2018-10-22 at 10 06 14 am" src="https://user-images.githubusercontent.com/3108007/47300466-241cea00-d5e2-11e8-9dd8-4880bb7bab33.png">

```css
  padding: 10px;
  ...
  background: #004b6b repeating-linear-gradient(
    135deg,
    transparent,
    transparent 20px,
    rgba(255, 255, 255, 0.1) 20px,
    rgba(255, 255, 255, 0.1) 40px
  );
```

Reply:
Yep, that looks better. Thanks @deckar01 .